pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -244,8 +291,27 @@ function Cite_replace(cite)
   end
 end
 
+local refsDivSeen=false
+function Div(div)
+  if not div.attr or div.attr.identifier ~= 'refs' then return nil end
+  refsDivSeen=true
+  if (not string.match(FORMAT, 'odt')) or config.csl_style == '' then return nil end
+  return pandoc.RawBlock('opendocument', zotero_bibl())
+
+end
+
+function Doc(doc)
+  if refsDivSeen then return nil end
+  if (not string.match(FORMAT, 'odt')) or config.csl_style == '' then return nil end
```

Comment:
This means it also gets added for scannable cite, is that intentional?

Reply:
> This means it also gets added for scannable cite, is that intentional?

For some reason I couldn't access 'config.format' at that point.  There is some Lua quirk that I am missing, possibly because format is _not_ declared. 

```
local config = {
  client = 'zotero',
  scannable_cite = false,
  csl_style=''
}
```

 If you are OK with adding format to that declaration,  then this jugglery here won't be needed.